src/terminal/input/terminalInput.cpp

```diff
@@ -576,6 +573,49 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
     return false;
 }
 
+// Method Description:
+// - Returns the keyboard's scan code for the given virtual key code.
+// Arguments:
+// - vkey: The virtual key code.
+// Return Value:
+// - The keyboard's scan code.
+WORD TerminalInput::_ScanCodeFromVirtualKey(const WORD vkey) noexcept
+{
+    return LOWORD(MapVirtualKeyW(vkey, MAPVK_VK_TO_VSC));
+}

```

Comment:
Part of me thinks we should move these helpers to some shared space, since this isn't the first time this has been written, but that's probably a better follow-up codehealth task.